src/Sylius/Bundle/AdminBundle/Resources/config/grids/product.yml

```diff
@@ -101,3 +109,6 @@ sylius_grid:
                 bulk:
                     delete:
                         type: delete
+    templates:
```

Comment:
Should be part of general grid config `src/Sylius/Bundle/AdminBundle/Resources/config/app/config.yml`